test/acceptance/accessibility.py

```diff
@@ -282,6 +281,26 @@ def test_multiple_ora_complete_flow(self):
         self._check_a11y(self.peer_asmnt_page)
 
 
+class FileUploadA11yTest(OpenAssessmentA11yTest):
+
+    def setUp(self):
+        super(FileUploadA11yTest, self).setUp('file_upload')
+
+    def test_file_upload(self):
+        self.auto_auth_page.visit()
+        self.submission_page.visit()
+        self._check_a11y(self.submission_page)
+
+    def _configure_a11y_audit_config(self, page):
+        super(FileUploadA11yTest, self)._configure_a11y_audit_config(page)
+        page.a11y_audit.config.set_rules({
+            "ignore": [
+                "color-contrast",  # TODO: TNL-5104
+                "list",  # TODO: TNL-5900
```

Comment:
Problem with the list structure-- created TNL-5900, which is a subtask of the more general "list structure is crazy" ticket.